scadalts-ui/src/apps/App.vue

```diff
@@ -133,11 +131,46 @@
 </template>
 
 <script>
+import webSocketMixin from '@/utils/web-socket-utils';
+import internetMixin from '@/utils/connection-status-utils';
+function reload () {
+	location.reload();
+}
 export default {
 	name: 'app',
+	mixins: [webSocketMixin, internetMixin],
 
 	data() {
-		return {};
+		return {
+			onAppOnline: () => {
+				this.wsLive = true;
+			},
+			onAppOffline() {
+				this.wsLive = false;
+			},
+			wsCallback: () => {		
+				this.wsLive = true;	
+				this.wsSubscribeTopic(`alarm`, async(x) => {
+					this.unsilencedHighestAlarmLevel = await this.$store.dispatch('getHighestUnsilencedAlarmLevel');
+				});		
+			},
+			wsLive: false,
+			unsilencedHighestAlarmLevel: -1,
```

Comment:
this variable should be defined in VuexStore because it can be mutated when user has "acknowledged the event" and then from VuexAxction it is much easier to manage the state of that element and reset that parameter.  

Now when user acknowledge all events the flag is still visible on the new User Interface even if in classic UI it was disposed.

Reply:
It's because the getHighestUnsilencedAlarmLevel api method is not considering ackowledged condition